src/fsharp/vs/Symbols.fs

```diff
@@ -1460,8 +1470,8 @@ and FSharpMemberOrFunctionOrValue(cenv, d:FSharpMemberOrValData, item) =
             let tau = v.TauType
             let argtysl,_ = GetTopTauTypeInFSharpForm cenv.g curriedArgInfos tau range0
             let argtysl = if v.IsInstanceMember then argtysl.Tail else argtysl
-            
-            [ for argtys in argtysl do 
+            let prettyArgtysl = argtysl |> List.map (fun a -> PrettyTypes.PrettifyTypesN1 cenv.g (a, tau)) 
```

Comment:
Not sure I understand, `PrettyTypes.PrettifyTypes1 tau` ?
Would we not have to do `GetTopTauTypeInFSharpForm` and then disregard the head if its an instance?